test-data/unit/check-flags.test

```diff
@@ -479,12 +479,13 @@ main:5: error: Constraint 2 becomes List[Any] due to an unfollowed import
 
 [case testDisallowImplicitAnyNewType]
 # flags: --ignore-missing-imports --disallow-any=unimported
-from typing import NewType
+from typing import NewType, List
 from missing import Unchecked
 
-Baz = NewType('Baz', Unchecked)
-[out]
-main:5: error: Argument 2 to NewType(...) must be subclassable (got Any)
+# this case is handled elsewhere: Baz = NewType('Baz', Unchecked)
```

Comment:
I would still include the test here. Tests shouldn't follow the implementation too closely -- they should tests features.